examples/openapi/README.md

```diff
@@ -9,39 +9,23 @@ This example will show you how to proxy the [Swagger Petstore](https://petstore3
 ### Running the example
 
 ```bash
-cargo run -- -f examples/openapi/config.json
+cargo run -- -f examples/openapi/config.yaml
 ```
 
-Let's look at the config to understand what's going on. First off we have a listener, which tells the gateway how to listen for incoming requests/connections. In this case we're using the `sse` listener, which is a simple HTTP listener that listens on port 3000.
-```json
-  "listeners": [
-    {
-      "sse": {
-        "address": "0.0.0.0",
-        "port": 3000
-      }
-    }
-  ],
-```
+In addition to the [basic](../basic) setup, we have added a new target of type `openapi`:
 
-Next we have a targets section, which tells the gateway how to proxy the incoming requests to the target. In this case we're proxying to the [Swagger Petstore](https://petstore3.swagger.io). In the future we will also support remote OpenAPI specs via URL.
-```json
-  "targets": {
-    "mcp": [
-      {
-        "name": "petstore",
-        "openapi": {
-          "host": "petstore3.swagger.io",
-          "port": 443,
-          "schema": {
-            "file_path": "examples/openapi/openapi.json"
-          }
-        }
-      }
-    ]
-  }
+```yaml
+name: openapi
+openapi:
+  schema:
+    file: ./examples/openapi/openapi.json
+  host: localhost
+  port: 8080
 ```
 
+This will expose each method in the openapi specification as MCP tools, and proxy them to the petstore application (on `localhost:8080`).
+
```

Comment:
Seems the images are outdated. I don't think that should hold up the PR. I can PR those later. But I noticed that in the inspector, it doesn't seem to understand the fields for the REST APIs that get exposed as tools. That is, I cannot enter any values in for the parameters.